### ISSUE TITLE
[FEATURE] Reimplement full width widgets

### ISSUE THREAD

**Issue by kobanogit** [Deletescape-Media/Lawnchair#4853]
### Describe the feature

I am using Lawnchair 14 (I used to use 12 before).
Display: Grid 6x6

In Lawnchair 14, widgets have a margin on the right and left that cannot be removed.
In Lawnchair 12, there was a "full width widget" option so that widgets occupy the entire width of the screen.

Would it be possible to put the "full width widget" option back?

Thanks a lot for this app and all its features !

### Why would it be useful to add?

This is especially useful for the KWGT widget that displays several images that are already very small, lookj at the screenshots.

### Did the feature exist in Lawnchair v2? (Play Store version)

Yes

### Example(s)

I don't know if this feature exist in Lawnchair v2.
![Lawnchair widget margins](https://github.com/user-attachments/assets/4755e0de-1c94-473b-a337-8770f1ec2ac9)

**Comment by daviessm** [comment]
Is this related to/fixed by #4879?

**Comment by kobanogit** [comment]
> Is this related to/fixed by #4879?

Yes, it seems to be the same request.
Is it possible to download the last commit and build the .apk ? (Sorry I'm a beginner)
Thank you !

**Comment by non-bin** [comment]
I don't think this is the same, the previous feature made essentially full screen widgets possible (I used that for my calendar) but this still leaves a pretty wide gap. Maybe this could work if we had per-page margin settings, but I think the previous implementation was better
